src/bootstrap/src/utils/exec.rs

```diff
@@ -24,14 +26,71 @@ pub enum OutputMode {
 }
 
 /// Wrapper around `std::process::Command`.
+///
+/// By default, the command will exit bootstrap if it fails.
+/// If you want to allow failures, use [allow_failure].
+/// If you want to delay failures until the end of bootstrap, use [delay_failure].
+///
+/// By default, the command will print its stdout/stderr to stdout/stderr of bootstrap
+/// ([OutputMode::OnlyOutput]). If bootstrap uses verbose mode, then it will also print the
+/// command itself in case of failure ([OutputMode::All]).
+/// If you want to handle the output programmatically, use `output_mode(OutputMode::OnlyOnFailure)`.
```

Comment:
I think we should say more about `OnlyOnFailure`; like "prints the output only if command fails".

Reply:
It will be nuked in the next PR if I remembers my commits :laughing: But I expanded the comment.